ProcessLib/GroundwaterFlow/Tests.cmake

```diff
@@ -682,6 +682,18 @@ AddTest(
     VIS square_1e6__nodal_sources_expected_pcs_0_ts_1_t_1.000000.vtu
 )
 
+AddTest(
+    NAME GroundWaterFlowProcess_VolumetricSourceTerm_square_1e2
+    PATH Elliptic/square_1x1_GroundWaterFlow
+    EXECUTABLE ogs
+    EXECUTABLE_ARGS square_1e2_volumetricsourceterm.prj
+    WRAPPER time
+    TESTER vtkdiff
+    REQUIREMENTS NOT (OGS_USE_LIS OR OGS_USE_MPI)
+    DIFF_DATA
+    square_1x1_quad_1e2_volumetricsourceterm_analytical_solution.vtu square_1e2_volumetricsourceterm_pcs_0_ts_1_t_1.000000.vtu analytical_solution pressure 1e-14 1e-16
```

Comment:
Is the solution a linear slope or a constant? OGS is extremely close to the analytical result.

Reply:
The analytical solution is h(x,y) = - 1/2 (x^2 + x) + 2.